src/element/textElement.ts

```diff
@@ -83,35 +83,25 @@ export const redrawTextBoundingBox = (
   boundTextUpdates.baseline = metrics.baseline;
 
   if (container) {
-    if (isArrowElement(container)) {
-      const centerX = textElement.x + textElement.width / 2;
-      const centerY = textElement.y + textElement.height / 2;
-      const diffWidth = metrics.width - textElement.width;
-      const diffHeight = metrics.height - textElement.height;
-      boundTextUpdates.x = centerY - (textElement.height + diffHeight) / 2;
-      boundTextUpdates.y = centerX - (textElement.width + diffWidth) / 2;
```

Comment:
this isn't needed anymore?

Reply:
no its taken care in `computeBoundTextPosition` hence removed from here

Reply:
for more context it was fixed in `computeBoundTextPosition` in this PR - https://github.com/excalidraw/excalidraw/pull/6445/files